platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/shadowgraph/ShadowGraphSynchronizer.java

```diff
@@ -174,10 +174,7 @@ public ShadowGraphSynchronizer(
         this.nonAncestorFilterThreshold = syncConfig.nonAncestorFilterThreshold();
 
         this.filterLikelyDuplicates = syncConfig.filterLikelyDuplicates();
-        this.latestEventTipsetTracker = latestEventTipsetTracker;
-        if (filterLikelyDuplicates) {
-            Objects.requireNonNull(latestEventTipsetTracker);
-        }
+        this.latestEventTipsetTracker = Objects.requireNonNull(latestEventTipsetTracker);
```

Comment:
`latestEventTipsetTracker` should now be annotated `@NonNull`

Reply:
the `latestEventTipsetTracker` data structure no longer exists